src/Sylius/Bundle/CoreBundle/Form/Type/Product/ChannelPricingType.php

```diff
@@ -35,7 +36,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $form = $event->getForm();
 
                 $form->add('price', MoneyType::class, [
-                    'label' => $channel->getName(),
+                    'label' => 'Price for ' . $channel->getName(),
```

Comment:
Yes, and the don`t plan to support this, so injecting the translator seems the only way to do it